docs/.vuepress/handsontable-manager/register.js

```diff
@@ -1,17 +1,17 @@
 /* eslint-disable no-restricted-globals, no-undef, no-console, no-unused-vars */
 
-const register = new Map();
+const register = new Set();
 
 register.listen = () => {
   try {
     if (typeof Handsontable !== 'undefined' && Handsontable._instanceRegisterInstalled === undefined) {
       Handsontable._instanceRegisterInstalled = true;
       Handsontable.hooks.add('afterInit', function() {
-        const hotId = this.rootElement
-          .closest('[hot-example-id]')
-          .getAttribute('hot-example-id');
-
-        register.set(hotId, this);
+        // Collect only HoT main instances. Skip context menu, dropdown menu
+        // and other internal HoT-based components.
+        if (this.rootElement.classList.contains('hot')) {
```

Comment:
AFAIK, we do not always set class 'hot' for a root element.

For instance, in the following files:
* searching-values.md
* batch-operations.md
* cell-validator.md
* react-simple-example.md

I may find `<div id="example1"></div>`

Will it still work properly for these examples?

Reply:
Good point! I fixed that within ee77382a7d7ef0b40f6790f935440156cf69910a. The instances prefixed by `ht_` are managed by the root HoT instance.